models/users.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const UserSchema = new Schema({
+  name: String,
+  username: { type: String, required: true, unique: true },
+  password: { type: String, required: true },
+  admin: Boolean,
```

Comment:
What is the admin field for? (If it is for permissions, it does not belong in the Users collection, and should be removed.)